### PR TITLE
added fdroid-badge

### PR DESCRIPTION
**Pull request by DJCrashdummy**
according to https://gitlab.com/fdroid/artwork/blob/master/badge/README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,6 +28,9 @@ components. It also showcases the usage of the contacts and calendar providers.
 <img alt="Screenshot 5" src="assets/device-art/device-phone-landscape-2.png" width="384" />
 
 ## Download
+[<img src="https://f-droid.org/badge/get-it-on.png"
+      alt="Get it on F-Droid"
+      height="80">](https://f-droid.org/app/saschpe.contactevents)
```
Comment (saschpe) [inline README.md]: The file preview didn't display the image. Rather use an  tag

Comment (DJCrashdummy) [reply README.md]: i had the same issue yesterday; seems the fdroid-server was kind of slow... as you can see at https://github.com/DJCrashdummy/BirthdayCalendar#download everything is fine now.

**Comment by saschpe** [comment]
Thanks!

**Comment by DJCrashdummy** [comment]
you're welcome @saschpe. thanks for your FOSS!

**Pull request merged**